corehq/apps/accounting/models.py

```diff
@@ -2451,6 +2494,31 @@ def get_credits_by_subscription_and_features(cls, subscription,
         ).all()
 
     @classmethod
+    def get_total_credits(cls, subscription=None, account=None, product_type=None, feature_type=None):
+        assert subscription or account
+        assert not product_type or not feature_type
+
+        kwargs = {}
+        if product_type:
+            kwargs['product_type'] = product_type
+        elif feature_type:
+            kwargs['feature_type'] = feature_type
+
+        credit_lines = []
```

Comment:
not sure this is needed, or you may have wanted to do `credit_lines.append()` in the if statements below.